content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -429,7 +429,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
   assigns a lowest possible score of 1 to a node that satisfies at least one of
   the input Pod's cpu and memory limits. The intent is to break ties between
   nodes with same scores.
-- `RequestManagement`: Enable managing request concurrency with prioritization and fairness at each server.
+- `APIPriorityAndFairness`: Enable managing request concurrency with prioritization and fairness at each server.
```

Comment:
@makocchi-git , Thanks for this PR.
Would you move the `APIPriorityAndFairness` description to around line 316?
The list is sorted alphabetically.
Also move the entry in the alpha/beta table to the first entry front in the table, around line 51.